sonar-db/src/test/java/org/sonar/db/favorite/FavoriteDbTester.java

```diff
@@ -58,4 +58,12 @@ public boolean hasFavorite(ComponentDto componentDto, long userId) {
 
     return !result.isEmpty();
   }
+
+  public boolean hasNoFavorite(ComponentDto componentDto) {
```

Comment:
you should provide a userId like `hasFavorite`. In fact if you really need a hasNoFavorite its implementation should only be: `return !hasFavorite(componentDto, userId);`